lib/ssl/test/tls_api_SUITE.erl

```diff
@@ -300,16 +300,16 @@ tls_upgrade_new_opts_with_sni_fun() ->
 tls_upgrade_new_opts_with_sni_fun(Config) when is_list(Config) ->
     ClientOpts = ssl_test_lib:ssl_options(client_rsa_verify_opts, Config),
     ServerOpts = ssl_test_lib:ssl_options(server_rsa_opts, Config),
-    ServerDsaOpts = ssl_test_lib:ssl_options(server_dsa_opts, Config),
+    ServerEdsaOpts = ssl_test_lib:ssl_options(server_ecdsa_opts, Config),
     {ClientNode, ServerNode, Hostname} = ssl_test_lib:run_where(Config),
     TcpOpts = [binary, {reuseaddr, true}],
     Version = ssl_test_lib:protocol_version(Config),
     NewVersions = new_versions(Version),
-    Ciphers =  ssl:filter_cipher_suites(ssl:cipher_suites(all, Version),
-                                        [{key_exchange, fun(srp_rsa) -> false;
-                                                           (srp_dss) -> false;
-                                                           (_) -> true
-                                                        end}]),
+    Ciphers = ssl:filter_cipher_suites(ssl:cipher_suites(all, Version),
+                                       [{key_exchange, fun(srp_rsa) -> false;
+                                                          (srp_dss) -> false;
+                                                          (_) -> true
+                                                       end}]),
```

Comment:
You do not need to specify ciphers  as you are no longer using an  RSA-key that may need old cipher-suites on old machines.